airbyte-integrations/connectors/source-opuswatch/manifest.yaml

```
@@ -0,0 +1,1496 @@
+            days_from_date: "366"
+            filter_date_by: CREATED
+            return_archived: "true"
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - data
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            field_name: offset
+            inject_into: request_parameter
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 10000
+        decoder:
+          type: JsonDecoder
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/sessions initial"
+    sessions incremental:
+      type: DeclarativeStream
+      name: sessions incremental
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: transactional/sessions
+          http_method: GET
+          request_parameters:
+            return_breaks: "true"
+            return_leaves: "true"
+            days_till_date: "2"
+            filter_date_by: UPDATED
+            return_archived: "true"
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - data
+        decoder:
+          type: JsonDecoder
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/sessions incremental"
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.opuswatch.nl/ext/
+    authenticator:
+      type: ApiKeyAuthenticator
+      api_token: "{{ config[\"api_key\"] }}"
+      inject_into:
+        type: RequestOption
+        field_name: key
+        inject_into: header
+
+streams:
+  - $ref: "#/definitions/streams/client"
+  - $ref: "#/definitions/streams/locations"
+  - $ref: "#/definitions/streams/rows"
+  - $ref: "#/definitions/streams/users"
+  - $ref: "#/definitions/streams/workers"
+  - $ref: "#/definitions/streams/worker groups"
+  - $ref: "#/definitions/streams/tasks"
+  - $ref: "#/definitions/streams/task groups"
+  - $ref: "#/definitions/streams/labels"
+  - $ref: "#/definitions/streams/varieties"
+  - $ref: "#/definitions/streams/registrations initial"
+  - $ref: "#/definitions/streams/registrations incremental"
+  - $ref: "#/definitions/streams/sessions initial"
+  - $ref: "#/definitions/streams/sessions incremental"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - api_key
+      - initial_data
+    properties:
+      api_key:
+        type: string
+        order: 0
+        title: API Key
+        airbyte_secret: true
+      initial_data:
```

```suggestion
      start_date:
```
You can make this a datetime field and later apply a transformation to this specific format.